doc/sphinx/source/notes_7_2.rst

```diff
@@ -0,0 +1,15 @@
+**************************
+Release Notes for PoCL 7.2
+**************************
+
+===================================
+Experimental and work-in-progress
+===================================
+
+* Added experimental MLIR-based OpenCL C kernel compilation flow for CPUs.
```

Comment:
```suggestion
* Added a work-in-progress MLIR-based OpenCL C kernel compilation flow for CPUs.
```

Reply:
Also mention which barrier scenarios are not supported to set the user expectation level.